com.unity.ml-agents/Runtime/SideChannels/EngineConfigurationChannel.cs

```diff
@@ -46,7 +46,15 @@ protected override void OnMessageReceived(IncomingMessage msg)
                     break;
                 case ConfigurationType.TimeScale:
                     var timeScale = msg.ReadFloat32();
-                    timeScale = Mathf.Clamp(timeScale, 1, 100);
+
+                    // There's an upper limit for the timeScale in the editor (but not in the player)
+                    // Always ensure that timeScale >= 1 also,
+#if UNITY_EDITOR
+                    const float maxTimeScale = 100f;
+#else
+                    const cloat maxTimeScale = float.PositiveInfinity;
+#endif
+                    timeScale = Mathf.Clamp(timeScale, 1, maxTimeScale);
```

Comment:
Could there be cases where someone wants to train at timescales between (0,1]?

Reply:
and do we want to support that?

Reply:
I can't imagine for training, but there could hypothetically be an inference use-case where someone might want it to go slower to better inspect the agent behavior.

Reply:
This block of code only affects timescale information coming from python (generally `mlagents-learn` commandline, but could be the yaml config or low-level API too), so inference isn't affected by this.

So far there haven't been any requests for timescale < 1, but we can change the lower bound to, say, 0.01.